docs/how-tos/run-graphql-query.md

```diff
@@ -0,0 +1,85 @@
+# How to Run A GraphQL Query
+
+This guide explains how to run a GRAPHQL query for workflows using the Workflows
+Dashboard or alternative tools.
+The Workflows Dashboard does provide a GraphiQL client to write queries to
+the GraphQL endpoint which is shown below.
+
+## Available Endpoints
+[Production](https://workflows.diamond.ac.uk/graphql)
```

Comment:
Is this user-facing documentation or internal documentation for Workflows developers?

Assuming that it is user-facing, should we be encouraging them to use the Workflows endpoints directly? Is it not better to encourage them to use the federated graph?